resources/js/components/data-list/Action.vue

```diff
@@ -104,6 +104,8 @@ export default {
         },
 
         confirm() {
+            this.confirming = false;
```

Comment:
This causes the dialog to close when you have a validation error.

Do you remember the reason for adding this?

Reply:
I'm just gonna remove it. If there's a problem we can fix it later.